src/modules/share/share.repository.ts

```diff
@@ -267,7 +263,7 @@ export class SequelizeShareRepository implements ShareRepository {
         },
       },
       include: [
-        this.userModel,
+        UserModel,
```

Comment:
Why are we including in a generic pagination the user model always? This slows all the use cases relying on this